spec/Private/CARFIT/Chassis/Wheel.vspec

```diff
@@ -12,7 +12,7 @@
   description: Brake pad wear status. True = Worn. False = Not Worn.
 
 - Brake.RotorWear:
-  type: Uint16
+  type: int16
```

Comment:
This changes from unsigned to signed.  Intentional?

Reply:
a mistake.... Will fix and push again (Same happened to driver heartrate...)